pkgs/development/python-modules/subunit/default.nix

```diff
@@ -1,22 +1,33 @@
 { buildPythonPackage
-, pkgs
+# pkgs dependencies
+, cppunit
+, pkg-config
+, check
+
+# python dependencies
+, subunit
```

Comment:
`subunit` is a pkgs dependency.

Reply:
Yea, looks when I was refactoring, some of my changes didnt occur. Thanks